src/main/java/com/microfocus/application/automation/tools/model/CredentialsModel.java

```diff
@@ -44,23 +44,23 @@
 
 public class CredentialsModel extends AbstractDescribableImpl<CredentialsModel> implements Serializable {
     private String almUsername;
-    private Secret almPassword;
+    private String almPassword;
```

Comment:
Sensitive data should not be stored as strings. See https://www.jenkins.io/doc/developer/security/secrets/.